torch/csrc/jit/ir/irparser.cpp

```diff
@@ -501,6 +501,13 @@ void IRParser::parseOperator(Block* b) {
   for (const VarWithType& v : outs) {
     vmap[v.name] = n->outputs()[idx];
     if (schema && !schema->is_varret()) {
+      TORCH_CHECK(
```

Comment:
curious, why do we need this check?

I understand the pop() issue... but isn't .at() already checked (i.e. this is why you use `.at()` instead of `[]`)?

Reply:
Yep, that's true.
However, I thought about this issue from the user's point of view. I'd say it's mostly a quality-of-life improvement that helps to clearly show where and why the error occurs. There are many places, where using `.at()` is enough, however, I think in this context it's beneficial.